### PR TITLE
Add instructions on configuring nginx-ingress with oauth2-proxy.

### PR DESCRIPTION
**Pull request by naseemkullah**

### CODE REVIEW

**Comment by grampelberg** [comment]
@cpretzer mind giving this a run through?

File: linkerd.io/content/2/tasks/exposing-dashboard.md [hunk 1]
```diff
@@ -28,29 +30,90 @@ metadata:
...
+A more secure alternative to basic auth is using an authentication proxy, such
+as [oauth2-proxy](https://github.com/pusher/oauth2_proxy).
+
+For reference on how to deploy and configure oauth2-proxy in kubernetes, see
+this [blog post by Don
+Bowman](https://blog.donbowman.ca/2019/02/14/using-single-sign-on-oauth2-across-many-sites-in-kubernetes/).
+
+Tl;dr if you deploy oauth2-proxy via the [helm
+chart](https://github.com/helm/charts/tree/master/stable/oauth2-proxy), the
+following value are required:
```
Comment (cpretzer) [inline linkerd.io/content/2/tasks/exposing-dashboard.md:67]: tl;dr should be all upper or lower case and "...the following value..." should be "... the following . values..."

Comment (naseemkullah) [reply linkerd.io/content/2/tasks/exposing-dashboard.md]: Thanks @cpretzer, done!

**Comment by naseemkullah** [comment]
bump

**Comment by grampelberg** [comment]
Whoops, I dunno why this didn't get merged. Thanks @naseemkullah !

**Comment by naseemkullah** [comment]
My pleasure @grampelberg!

**Pull request merged**